examples/input/touch_input.rs

```diff
@@ -0,0 +1,23 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::build()
+        .add_default_plugins()
+        .add_system(touch_system.system())
+        .run();
+}
+
+// This system prints messages when you use the touchscreen
+fn touch_system(finger_input: Res<Input<Finger>>) {
+    if finger_input.pressed(Finger(0)) {
+        println!("finger 0 pressed");
```

Comment:
I'm not 100% sure this works on all platforms? For example on iOS I was getting huge numbers, not 0 and 1.

Reply:
Yeah I think we should assume that a "touch id" is an opaque identifier.